### ISSUE TITLE
Screensharing support

### ISSUE THREAD

**Issue by eporsche** [ivelkov/teams-for-linux#43]
Hi,
this software works flawless :) - except Screensharing does not work. 
Are there plans to integrate this functionality?
Thanks!

**Comment by mpruefer** [comment]
Yes, the ability to screensharing is the only thing that’s missing really. Except this, great app!

**Comment by jesse-osiecki-zz** [comment]
I'm not sure this is possible, it sounds like they only offer it on the desktop client?

**Comment by eporsche** [comment]
I guess you are right - I tried Screen Sharing with Edge under Windows. It does not work. Seems like a desktop client only feature.

**Comment by shanalikhan** [comment]
Missing screen share feature

**Comment by waynedmoore** [comment]
Screen sharing is not possible in the web client, which is what this is based on. So I'd say this would not be possible until there is a native Linux client - whenever that may be.

**Comment by eporsche** [comment]
I was playing around with  [v4l2loopback]( https://github.com/umlaeute/v4l2loopback) .. it works - but with very bad quality...  I ran the loopback device with ` ffmpeg -f x11grab -r 15 -s 1280x720 -i :1 -vcodec rawvideo -pix_fmt yuv420p -threads 0 -vf hflip -f v4l2 /dev/video1 `

**Comment by noderat** [comment]
Appears there is now a chrome extension for screen sharing in teams?

https://chrome.google.com/webstore/detail/microsoft-teams-screen-sh/dhheiegalgcabbcobinipgmhepkkeidk?hl=en

**Comment by KirylKovaliov** [comment]
I'm also experiencing issue with the direct calls. If i call to the group everything works fine, but 1 on 1 it doesn't work at all.

**Comment by brennerm** [comment]
Did anybody get the extension to work with the Electron app?

**Comment by julian-alarcon** [comment]
Check this fork and star it! https://github.com/IsmaelMartinez/teams-for-linux It is currently supported!
Screen sharing was enabled since version 0.1.16.
